src/v/cluster/types.h

```diff
@@ -498,6 +498,11 @@ incremental_update_operation_as_string(incremental_update_operation op) {
     }
 }
 
+inline std::ostream&
+operator<<(std::ostream& os, const incremental_update_operation& op) {
+    return os << incremental_update_operation_as_string(op);
+}
```

Comment:
nit: Once we have this operator, can we omit `incremental_update_operation_as_string` function call in `operator<<(std::ostream& o, const property_update<T>& p)` and `operator<<(std::ostream& o, const property_update<tristate<T>>& p)` to get rid of this function?